package.json

```diff
@@ -23,7 +23,7 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-subset": "^1.6.0",
-    "codeceptjs": "^2.6.10",
+    "codeceptjs": "^3.0.0-beta.3",
```

Comment:
do you want to wait for it to go GA?

Reply:
Alright @frickjack , looks like they published `3.0.1` 😄 So, I will run a full set of tests, merge and keep a close eye on the pipeline.